src/App.tsx

```diff
@@ -8,7 +8,67 @@ import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
+import { Todo } from './types/Todo';
+import { FilterType } from './types/FilterType';
+import { getTodos } from './api';
+
+const filteredTodos = (todos: Todo[], inputQuery: string, filterType: FilterType) => {
```

Comment:
The function name is the action, not the result of the action. The result of the performed action is the name for the variable
```suggestion
const filterTodos = (todos: Todo[], inputQuery: string, filterType: FilterType) => {
```